doc/movingparts.rst

```diff
@@ -90,15 +86,14 @@ The serializer lets you write HTML back as a stream of bytes.
   '>'
   'Witam wszystkich'
 
-You can customize the serializer behaviour in a variety of ways, consult
-the :class:`~html5lib.serializer.htmlserializer.HTMLSerializer`
-documentation.
+You can customize the serializer behaviour in a variety of ways. Consult
+the :class:`~html5lib.serializer.HTMLSerializer` documentation.
 
 
 Filters
 ~~~~~~~
 
-You can alter the stream content with filters provided by html5lib:
+html5lib provides several filters
```

Comment:
Given that what follows is a bulleted list, can you add a `:` to the end of this line?